getting-started/singleframe.rst

```diff
@@ -0,0 +1,164 @@
+If you expect that all of the dataset types should already be registiered, as is the case when processing another subset of data with a pipeline that has already been run, it can help catch unexpected behavior to remove that switch.
+
+.. tip::
+   It is not included in the above command, but the ``-j`` option is useful if you have more than one core available to you.
+   Specifying ``-j #`` will run in parallel where ``#`` is the number of processes to execute in parallel.
+
+   `Dataset queries`_ can be specified using the ``-d`` argument to specify which specific datasets should be considered when building the execution graph.
+   If this argument is omitted, all data in the repository that can be processed based on other inputs, e.g. calibrations, will be.
+
+Aside: collections and quantum graphs
+============================================
+
+`Collections`_ are the primary way data in butler repositories are organized.
+Of the types of collections available, the two of interest here are the ``RUN`` and ``CHAINED`` types.
+
+``RUN`` collections are the least flexible.
+Once a dataset is added to a ``RUN`` collection, it can never be removed.
+The constraints on datasets in ``RUN`` collections makes these collections that most efficient to store and query.
+The collection containing the raw data is a ``RUN`` collection.
+
+``CHAINED`` collections are groupings of other collections associated with an alias for that grouping.
+The grouping of collections defines the order of collections to search when looking for a dataset associated with a specific data ID.
+The collection produced from the ``-o`` option above is a ``CHAINED`` collection.
+The ouput collection will generall include all the collections in the input plus any ``RUN`` collections produced by the processing.
+
+The first step of process data is to produce the quantum graph for the processing.
+This is a directed acyclic graph that completely defines inputs and outputs for every node (quantum) in the graph.
+As seen above, the quantum graph can be visualized to see how the processing is intended to progress.
+
+Wrap up
+=======
+
+In this tutorial, you've used the :command:`pipetask run` command to calibrate ``raw`` images in a Butler repository.
+Here are some key takeaways:
+
+- The :command:`pipetask run` command, with appropriate arguments and switches, processes ``raw`` datasets, applying both photometric and astrometric calibrations.
+- Datasets are described by both a *type* and *data ID*.
+  Data IDs are key-value pairs that describe a dataset (for example ``filter``, ``visit``, ``ccd``, ``field``).
+- `Dataset queries`_ can be used to specify which datasets to process..
+- Pipelines write their outputs to a Butler data repository.
+  Collections are used to organize and associate outputs of processing with the inputs to the processing.
+
+Continue this tutorial in :doc:`part 3, where you'll learn how to display these calibrated exposures <display>`.
```

Comment:
I do wonder if there is a path through this tutorial where you haven't just spent 11 hours  to get through this section but instead have a faster route that says "use the following constraint to the query to be on the fast track".  Obviously the fast track will not let them run ubercal but it will surely teach them something.

Reply:
As I said before, I'd prefer to keep the branching to a minimum for now since I'm just trying to translate what was there before.  I'll make a ticket to put in a tip box to show how to make faster headway